widgetastic_manageiq/__init__.py

```diff
@@ -4546,6 +4559,22 @@ def collapse_details(self, map_name):
             raise ItemNotFound('Collapse button not present,'
                 'map clusters/networks/datastores/associated_plans details is not expaded.')
 
+    def delete_mapping(self, map_name, cancel=False):
+        try:
+            el = self._get_map_element(map_name)
+            self.browser.click(self.ITEM_DELETE_TRASH_ICON_LOCATOR, parent=el)
+            # below root_browser is required as the buttons below are on modal prompt
+            # browser with parent=el won't work here.
+            if not cancel:
+                del_btn = self.root_browser.element(self.ITEM_PROMPT_DELETE_BUTTON_LOCATOR)
+                del_btn.click()
+            else:
+                cancel_btn = self.root_browser.element(self.ITEM_PROMPT_CANCEL_BUTTON_LOCATOR)
```

Comment:
please remove btn from cancel_btn.

Reply:
just to keep consistent with `del_btn`, keeping `btn` at the end.